index.html

```diff
@@ -241,6 +241,57 @@ <h3>Skills</h3>
 
     </section>
 
+    <!----------------- form area -->\
+    <section id="form-area1">  
+       
+        <div class="txt">
+            <P class="form-txt">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </P>
+        </div>
+
+              <form class="form-area" action="https://formspree.io/f/mjvdwrry" method="POST"> 
+                <form action="https://formspree.io/f/{form_id}" method="post">
+                <div class="user-details">
+                    
+                 
+                            <input type="text" placeholder="First name" required>
+                       
+                            <input type="email" placeholder="Email address"  pattern=".+@globex\.com" size="30" required >
+    
+                            <textarea name="Message" id="message" placeholder="Write me something..." id="text-box" type="text"  maxlength="500"></textarea>
+                
```

Comment:
- [ ] As requested by the first reviewer, all the form elements should have the attribute `required`. Kindly implement this change because it is the required HTML validation for this element in this milestone. 👍 😎

Reply:
hi [Gambit142](https://github.com/Gambit142)
it resolved now

Reply:
Hi @techmoves, great work making the changes. I think it is alright. You can request another review. 👍